src/index.html

```diff
@@ -4,10 +4,263 @@
+            alt="#"
+            >
+          </div>
+          <h3 class="title title--information">Range & Integrated lights</h3>
+          <p class="article__paragraph">
+            The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+          </p>
+        </article>
+
+        <article class="article">
+          <div class="pictures  pictures--another">
+            <img
+            class="pictures__item pictures__item--left-smaller"
+            src="./images/photos/image5.jpg"
+            alt="#"
+            >
+            <img
+            class="pictures__item pictures__item--right-bigger"
+            src="./images/photos/image6.jpg"
+            alt="#"
+            >
+          </div>
+          <h3 class="title title--information">Hydraulic disc brakes & Lightweight</h3>
+          <p class="article__paragraph">
+            Brakes with total stopping power the second you make contact. The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+          </p>
+        </article>
+
+        <a href="#" class="button button--explore">
+          Explore
+        </a>
+      </section>
+
+      <!-- Tutaj inputy +  dane kontaktow-->
+
+      <h2 class="title  title--section" id="contact-us">Contact us</h2>
+      <section class="section  main__section main__section--3">
+        <div class="wrapper section__grid2">
+          <form class="form section__form">
+            <div class="form__item form__item--first">
+              <label for="name" class="form__label">Name</label>
+              <input
+                id="name"
+                type="text"
+                class="form__input"
+                placeholder="Name"
+                required
+              >
+            </div>
+
+            <div class="form__item">
+              <label for="email" class="form__label">Email</label>
+              <input
+              id="email"
+              type="email"
+              class="form__input"
+              placeholder="Email"
+              required
+            >
+            </div>
+
+            <div class="form__item">
+              <label for="message" class="form__label">Message</label>
+              <textarea
+              id="message"
+              class="form__textarea"
+              placeholder="Message"
+              required
+              ></textarea>
+            </div>
+            <button
+            class="button button--primary"
+            type="submit"
+            id="button-primary">
+              Send
+            </button>
+          </form>
+
+          <address class="address section__adress"> <!--Pomyślec o dobraniu w pary-->
+            <h4 class="address__label">
+              Phone
+            </h4>
+            <p class="address__content" id="number">
```

Comment:
why do you need additional p for links? can't you just style `<a>` element? 
it's a good practice to avoid html elements where they are not needed to prevent excessive-DOM size